File: Kyu 8-6/GO/SumOfNumbers.go
package kata

func MinMax(a,b int) (int,int) {
  if a < b {
    return a,b;
  } else {
    return b,a;
  }
}

func GetSum(a, b int) int {
  min,max := MinMax(a,b);
  count := 0;
  for min <= max {
    count+=min;
    min++;
  }
  return count;
}

File: Kyu 8-6/JavaScript - TypeScript/SumOfNumbers.js
function getSum( a,b )
{
   if(a == b) {
     return a;
   }
  // Usar Math.min e Math.max também funciona
  const sort = [a,b].sort((a,b) => a == b ? 0 : a > b ? -1 : 1);
  let min = sort[1];
  const max = sort[0]
  let count = 0;
  for(min; min <= max;min++) {
    count+=min;
  }
  return count;
}